8.frontend-react/managerapp/src/Components/Form/css/loginForm-style.css

```diff
@@ -0,0 +1,41 @@
+.loginForm {
```

Comment:
read about the advantages of responsive design system with this example:

talk with me.


// functions
@function lower-bound($range){
    @if length($range) <= 0 {
      @return 0;
    }
    @return nth($range,1);
  }
  
  @function upper-bound($range) {
    @if length($range) < 2 {
      @return 999999999999;
    }
    @return nth($range, 2);
  }

// Defining values
$iphone-range: (375px, 414px); 

$android-range: (360px,411px);

$mobile-range : (360px,450px);

$tablet-range: (768px, 1024px);

$desktop-range:(1366px,1920px);


// Defining media queries
$screen:       "screen" !default;
$landscape:    "#{$screen} and (orientation: landscape)" !default;
$portrait:     "#{$screen} and (orientation: portrait)" !default;

$iphone-range-up:    "#{$screen} and (min-width:#{lower-bound($iphone-range)})" !default;
$iphone-range-only:  "#{$screen} and (min-width:#{lower-bound($iphone-range)}) and (max-width:#{upper-bound($iphone-range)})" !default;

$android-range-up:    "#{$screen} and (min-width:#{lower-bound($android-range)})" !default;
$android-range-only:  "#{$screen} and (min-width:#{lower-bound($android-range)}) and (max-width:#{upper-bound($android-range)})" !default;

$tablet-range-up:    "#{$screen} and (min-width:#{lower-bound($tablet-range)})" !default;
$tablet-range-down:    "#{$screen} and (max-width:#{upper-bound($tablet-range)})" !default;
$tablet-range-down-small:    "#{$screen} and (max-width:#{lower-bound($tablet-range)})" !default;
$tablet-range-only:  "#{$screen} and (min-width:#{lower-bound($tablet-range)}) and (max-width:#{upper-bound($tablet-range)})" !default;

$mobile-range-up:    "#{$screen} and (min-width:#{lower-bound($mobile-range)})" !default;
$mobile-range-down:  "#{$screen} and (max-width:#{upper-bound($mobile-range)})" !default;
$mobile-range-only:  "#{$screen} and (min-width:#{lower-bound($mobile-range)}) and (max-width:#{upper-bound($mobile-range)})" !default;

$desktop-range-up:    "#{$screen} and (min-width:#{lower-bound($desktop-range)})" !default;
$desktop-range-only:  "#{$screen} and (min-width:#{lower-bound($desktop-range)}) and (max-width:#{upper-bound($desktop-range)})" !default;


/* Usage : 
    @media #{$iphone-range-up}     { ... }
    @media #{$android-range-only}     { ... }
    @media #{$tablet-range-up}     { ... }
    @media #{$mobile-range-down}     { ... }
    @media #{$desktop-range-up}     { ... }
*/